cfme/common/vm.py

```diff
@@ -298,8 +298,13 @@ def find_quadicon(
         # paginator.results_per_page(1000)
         if use_search:
             if not search.has_quick_search_box():
-                sel.force_navigate('vm_templates_provider_branch',
-                                   context={'provider_name': self.provider.name})
+                # We don't use provider-specific page (vm_templates_provider_branch) here
+                # as those don't list archived/orphaned VMs
+                if self.provider.category == "infra":
+                    sel.force_navigate('infra_vm_and_templates')
```

Comment:
This should a be a string that is set in the base provider classes in the cloud and infra modules.